driver/bpf/filler_helpers.h

```diff
@@ -606,8 +606,7 @@ static __always_inline int unix_socket_path(char *dest, const char *user_ptr, si
 	 * specified length of the address structure.
 	 */
 	if(res == 1) {
-		dest[0] = '@';
-		res = bpf_probe_read_kernel_str(dest + 1,
+		res = bpf_probe_read_kernel_str(dest,
```

Comment:
in the modern we don't prepend the `@` so it's probably better to be uniform between drivers, we can add it in the modern if we want but not sure what is the value of having the initial `@`